packages/flodash/test/modules/string/to_upper_test.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flodash/flodash.dart' as flodash;
+
+void main() {
+  test('toUpper should convert the string to upper case', () {
```

Comment:
The toUpper prefix in front of the test isn't needed ne ? the tests are already in scope due to them being in a separate file